src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -7,16 +7,16 @@
 from pathlib import Path
 from typing import Any, Dict, List, Union
 
-from sphinx.application import Sphinx
+from sphinx.application import Sphinx  # noqa: I003
 from sphinx.util.docutils import SphinxDirective
 from sphinx.util.nodes import nested_parse_with_titles, nodes
 
 
 # isort: split
 
-from docutils import statemachine
+from docutils import statemachine  # noqa: I001
```

Comment:
I think it's better to disable `I` via `extend-ignore` for now. It generates a lot of false-positives anyway.